shared/js/background/chrome-events.es6.js

```diff
@@ -389,12 +389,13 @@ chrome.runtime.onMessage.addListener((req, sender, res) => {
             const parsed = tldts.parse(tabUrl)
             action.tabRegisteredDomain = parsed.domain === null ? parsed.hostname : parsed.domain
 
+            action.isThirdParty = !utils.isFirstParty(sender.url, sender.tab.url)
+            action.shouldBlock = !cookieConfig.isExcluded(sender.url)
+
             if (req.documentUrl && trackerutils.isTracker(req.documentUrl) && sender.frameId !== 0) {
-                action.isTrackerFrame = true
+                action.isTrackerFrame = true && action.shouldBlock
```

Comment:
This change seems unrelated to the list caching. Did you mean to include it?

Reply:
I did. The way the content script was set up was ignoring exclusions. This will check if the url is on the exclusion list then apply it to `isTrackerFrame` accordingly. Google login for slack was broken before this change.

Reply:
Ok, rather than changing the semantics of `isTrackerFrame`, can you maybe change the check here to not inject if `shouldBlock` is false? https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/develop/shared/js/content-scripts/block-cookie.js#L132